apisix/init.lua

```diff
@@ -640,6 +641,14 @@ function _M.http_access_phase()
             enable_websocket = service.value.enable_websocket
         end
 
+        if route.value.strip_path_prefix and service.value.path_prefix then
+            local uri = api_ctx.var.uri
+            if uri:find(service.value.path_prefix) == 1 then
```

Comment:
Why not using `core.string.has_prefix`?

Reply:
lua built-in string.find is more simple and high efficient.